versioned_docs/version-1.2.x/surrealql/statements/create.mdx

```diff
@@ -6,7 +6,7 @@ sidebar_position: 6
 
 # `CREATE` statement
 
-The CREATE statement can be used to add records to the database, if those records do not already exist.
+The `CREATE` statement can be used to add records to the database, if those records do not already exist. In the case that the record already exists, the statement will fail.
```

Comment:
We can't actually insert many records with one create statement (but we can with insert) therefore I changed it to be singular here. Does that make sense?

Reply:
Yes! Makes sense. When updating the insert documentation I can add that there.